### PR TITLE
UI/final check

### PR DESCRIPTION
**Pull request by tsato21**
【UI/finalCheck】Final modifications on UI, links, and  routes
- Modified styles
- Modified routes in blade files
- Included modal file paths in show files

### CODE REVIEW

File: resources/views/admin/statistics/show.blade.php [hunk 1]
```diff
@@ -1,7 +1,18 @@
 @extends('layouts.admin')
 @section('title', 'Admin:Statistics')
 @section('content')
+
 <link href="{{ asset('css/style.css') }}" rel="stylesheet">
+{{-- CDN for jQeury (NOT USED)--}}
+{{-- <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script> --}}
```
Comment (kredo-school) [inline resources/views/admin/statistics/show.blade.php]: Remove commented codes

File: app/Http/Controllers/admin/StatisticsController.php [hunk 2]
```diff
@@ -1,14 +1,4 @@
...
-namespace App\Http\Controllers\Admin;
-
-include __DIR__ . '/../../../../resources/views/admin/statistics/SampleData.php';
-
-/* Following namespace import statement will be used at the phase of backend
-use Apps\Models\User;
-use Apps\Models\Reservation;
-use Apps\Models\Attribute;
-*/
 namespace App\Http\Controllers\admin;
```
Comment (kredo-school) [inline app/Http/Controllers/admin/StatisticsController.php:3]: Please remove
include __DIR__ . '/../../../../resources/views/admin/statistics/SampleData.php';

File: resources/views/admin/statistics/show.blade.php [hunk 3]
```diff
@@ -1,7 +1,18 @@
...
+{{-- Downloaded jQuery --}}
+<script type="text/javascript" src="{{ asset('js/jquery-3.7.1.min.js')}}"></script>
+
+{{-- CDN for Chart.js --}}
+{{-- <script src="https://cdn.jsdelivr.net/npm/chart.js"></script> --}}
```
Comment (kredo-school) [inline resources/views/admin/statistics/show.blade.php]: Remove commented codes

File: app/Http/Controllers/admin/StatisticsController.php (hunk 2 above)
Comment (tsato21) [reply app/Http/Controllers/admin/StatisticsController.php:3]: @kredo-school 
Thank you for your suggestion, Carlo-san.

As for this file path, let me keep it unitil the backend dev is complete.
Although I tried to reference the sample file in a consice way, I could not find it at the time that I made this file.

File: resources/views/admin/statistics/show.blade.php (hunk 1 above)
Comment (tsato21) [reply resources/views/admin/statistics/show.blade.php]: Done

File: resources/views/admin/statistics/show.blade.php (hunk 3 above)
Comment (tsato21) [reply resources/views/admin/statistics/show.blade.php]: @kredo-school 
Done

**Pull request merged**